config/config.js

```diff
@@ -28,8 +28,32 @@ const config = {
         database: 'test'
     },
 
+    VALIDATOR: {
+        /**
+         * Add custom validators here
+         * Format: { "validatorName": function(value, [additional arguments]), ... }
+         */
+        customValidators: {
```

Comment:
I suggest we try it first on one of the projects to have a test if it's easier to work with. Same as what we did with app-module-path, _.merge, and logging of db host on start.